raiden/raiden_event_handler.py

```diff
@@ -318,6 +338,7 @@ def on_raiden_event(raiden: RaidenService, event: Event):
         # handle_contract_send_channelclose2(raiden, event)
     elif type(event) == ContractSendChannelUpdateTransfer:
         handle_contract_send_channelupdate(raiden, event)
+        # handle_contract_send_channelupdate2(raiden, event)
```

Comment:
This is there just to remind you later that you should use the number2 so that you swap them?

Reply:
Yes, I have always added the function + the location where it is supposed to be used. Agreed, in this case it would have been clear anyway..